### PR TITLE
Added non-profit status and EIN information.

### PR DESCRIPTION
**Pull request by ktp242**
This PR adds the information of non-profit and EIN to the footer. Related ticket is here,
https://jira.nypl.org/browse/DQ-213

Please copy all the content in `footer.html` to the HTML section of `https://codepen.io/edwinguzman/pen/pKoWxq` and you should be able to see the updates under the copyright.

### CODE REVIEW

File: footer.html [hunk 1]
```diff
@@ -112,7 +112,8 @@
     </ul>
   </div>
   <div class="copyright">
-    <p>© The New York Public Library, 2018</p>
+    <p>© The New York Public Library, 2019</p>
```
Comment (EdwinGuzman) [inline footer.html:115]: Ha this should maybe be js so it can update automatically.

**Review by EdwinGuzman — APPROVED** [review]

File: footer.html (hunk 1 above)
Comment (ktp242) [reply footer.html:115]: yeah! I can make that.

Comment (EdwinGuzman) [reply footer.html:115]: I would not bother with that change in case this gets updated this year along with the website redesign.

Comment (ktp242) [reply footer.html:115]: ha OK then!

**Pull request merged**